features/step_definitions/role.js

```diff
@@ -116,7 +116,7 @@ var apiSteps = function () {
       return callback('Fixture for role ' + role + ' does not exist');
     }
 
-    index = Object.keys(this.roles[role].controllers)[0];
+    // todo : This test seams to have been wrongly adapted, controller variable is undefined
```

Comment:
My bad, I did not refactored properly this test after the role refactoring.
Actually, we should keep the deleted line, just replacing `index =` by `controller =`:
```
controller = Object.keys(this.roles[role].controllers)[0]; 
```